pkg/urlutil/git.go

```diff
@@ -8,11 +11,13 @@ var (
 		"github.com/",
 		"git@",
 	}
+
+	urlPathWithFragmentSuffix = regexp.MustCompile(".git(:?#.+)?$")
```

Comment:
I'm trying to parse this regexp.... I think this allows for:
```
.git:#abc
```
what that the intent?  If so, what does the ":" do and we should probably show it in the table.